src/components/TableCard/TableCard.test.jsx

```diff
@@ -54,6 +54,11 @@ describe('TableCard', () => {
     );
     expect(zeroMatchingThreshold2).toHaveLength(0);
   });
+  test('createColumnsWithFormattedLinks adds renderDataFunction to columns with links', () => {
+    const columnsWithFormattedLinks = createColumnsWithFormattedLinks(tableColumnsWithLinks);
+    const columnsWithlinks = columnsWithFormattedLinks.filter(column => column.renderDataFunction);
+    expect(columnsWithlinks).toHaveLength(1);
+  });
```

Comment:
It would be good to add another test with `@testing-library/react` that uses the new function to render to the DOM and then assert that the rendered output is a link as intended.

Reply:
Hey @tay1orjones, I have updated the story for TableCard. Notice that the links in the 'Link' column are populated based on the scope of that row.
You can change change the variable (to 'count' for example) to see that the link will update the endpoint based on what variable it has. 
I have also added a RTL test case to show that the link will be updated with its row specific variable value.

Reply:
Awesome, thanks - looks great!